terraform/modules/gost_website/waf.tf

```diff
@@ -23,5 +23,10 @@ module "waf" {
 
   managed_rule_group_statement_rules = local.expanded_rules
 
+  # WAFv2 must be managed in us-east-1 if globally scoped
+  providers = {
+    aws = aws.cloudfront_certificate
```

Comment:
Since I'm re-using this now, should we consider renaming it to something less specific? @TylerHendrickson

Reply:
Yeah, good call – maybe setting the alias to something like `us-east-1` (I think `-` characters are okay? can use underscores if not) would be more appropriate.

Would you also mind moving that aliased `provider` definition block from `certificate.tf` to `main.tf` in this module?